src/convert/types.ts

```diff
@@ -64,6 +64,7 @@ export type MergeConfig = {
    * Location to store components that aren't merged.
    */
   defaultDirectory: SourcePath;
+  forceIgnoredPaths?: string[];
```

Comment:
Can we use a [Set](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set) for ignoredPaths? `Array.prototype.includes` is an O(n) operation while `Set.prototype.has` is amortized constant time.

Reply:
Will do, thanks Xiaoyi!